SU-SWS/stanford_profile
# D8 core 1028 line length

# READY FOR REVIEW

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-1028_

# Needed Review By (Date)
- Sprint end

# Urgency
- Fixes broken stuff

# Steps to Test

- Checkout this branch
- Create lines of text with the appropriate lengths, and with one character more and one character less
- Verify the line lengths match:
```
H1
Leave as is

H2
56 characters

H3
60 characters

H4
64 characters

H5
76 characters

H6
84 characters

Intro
70 characters

Splash
Leave as is

Quote
54 characters

Callout
60 characters

Subhead
76 characters
```

# Affected Projects or Products 
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# QA
## HTML Validation Results (https://validator.w3.org/)
n/a

## Accessibility Testing 
N/A

## Cross-Browser testing:
What browsers have been tested?
- Chrome Desktop - Yes
- Chrome Android
- FireFox Desktop
- FireFox Android
- OSX Safari
- iOS Safari
- Edge
- IE 11

## Unit Testing
N/A

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-1028
## Related PRs


## More Information
It might make sense to put this up for Kerri to review. 

Here is data for testing:

[TestLineLength.txt](https://github.com/SU-SWS/stanford_profile/files/4161587/TestLineLength.txt)


## Folks to notify
`@sherakama` 
@kerri-augenstein

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
